### PR TITLE
Boost: Critical CSS and other small fixes

### PR DESCRIPTION
**Pull request by dilirity**
## Proposed changes:

* Move styles from global critical-css.scss file to their components (remove ones that aren't used);
* Update JS and CSS files to use path aliases;
* Remove unnecessary spacing below Critical CSS status section (when it's last).

<details><summary>Spacing before</summary>

CleanShot 2024-01-22 at 12 57 32

</details>

<details><summary>Spacing now</summary>

CleanShot 2024-01-22 at 12 58 47

</details>

### Other information:

- [ ] Have you written new tests for your changes, if applicable?
- [ ] Have you checked the E2E test CI results, and verified that your changes do not break them?
- [ ] Have you tested your changes on WordPress.com, if applicable (if so, you'll see a generated comment below with a script to run)?

## Jetpack product discussion

https://github.com/Automattic/jetpack/issues/35010

## Does this pull request change what data or activity we track or use?

no

## Testing instructions:

* Setup Boost;
* Make sure UI isn't broken.

### CODE REVIEW

File: projects/plugins/boost/app/assets/src/js/pages/critical-css-advanced/critical-css-advanced.tsx [hunk 1]
```diff
@@ -61,7 +61,7 @@ export default function AdvancedCriticalCss() {
 	};
 
 	return (
-		<div className="jb-container--narrow jb-critical-css__advanced">
```
Comment (thingalon) [inline projects/plugins/boost/app/assets/src/js/pages/critical-css-advanced/critical-css-advanced.tsx]: This class is used in the E2E test at `tests/e2e/lib/pages/wp-admin/JetpackBoostPage.js`. Can you please review whether this needs to be kept here for the test, or if the test needs updating?

**Review by thingalon — CHANGES_REQUESTED** [review]

File: projects/plugins/boost/app/assets/src/js/pages/critical-css-advanced/critical-css-advanced.tsx (hunk 1 above)
Comment (dilirity) [reply projects/plugins/boost/app/assets/src/js/pages/critical-css-advanced/critical-css-advanced.tsx]: Nice catch! I keep forgetting about e2e tests...

I've added the class back in so tests should be happy again.

**Review by thingalon — APPROVED** [review]
Thanks for fixing the e2e bug :)

**Comment by thingalon** [comment]
Cherry-picked to 3.0.0 in cd95296969.

**Pull request merged**